src/mainwindow.h

```diff
@@ -115,6 +115,7 @@ public slots:
     void mouseReleaseEvent(QMouseEvent *event) Q_DECL_OVERRIDE;
     void mouseDoubleClickEvent(QMouseEvent *event) Q_DECL_OVERRIDE;
     void leaveEvent(QEvent *) Q_DECL_OVERRIDE;
+    void changeEvent(QEvent *event) Q_DECL_OVERRIDE;
```

Comment:
Note for future patch, we should replace all `Q_DECL_OVERRIDE` with just `override`, since we're relying on C++14 anyway.

Reply:
Meh, that seems like an trivial change I could easily include here...